scripts/releaseCommon.py

```diff
@@ -128,13 +128,13 @@ def updateConanTestFile(version):
         f.write( line + "\n" )
 
 def updateCmakeFile(version):
-    cmakeParser = re.compile(r'set(CATCH_VERSION_NUMBER \d+\.\d+\.\d+)')
+    cmakeParser = re.compile(r'project(Catch2 LANGUAGES CXX VERSION \d+\.\d+\.\d+)')
```

Comment:
This line can be just deleted completely -- it stopped being used at some point.